dcc-release-core/src/main/java/org/icgc/dcc/release/core/task/GenericTask.java

```diff
@@ -249,6 +275,11 @@ protected void writeOutput(JavaRDD<ObjectNode> processed, String outputPath, boo
     return HadoopFiles.sequenceFileWithKey(sparkContext, path, conf, clazz);
   }
 
+  private static List<Path> resolveInputPaths(TaskContext taskContext, Path fileTypePath) {
+    return HadoopUtils.lsDir(taskContext.getFileSystem(), fileTypePath,
+        Pattern.compile(Partitions.PARTITION_NAME + ".*"));
```

Comment:
Assign to constant to name it for better understanding (and also for a _tiny bit_ of performance).